### ISSUE TITLE
stuck at starting telemetry

### ISSUE THREAD

**Issue by tikot** [ceejbot/aerogel#17]
Having an issue where it will not continue it gets stuck at:

```
STATE: setup --> connected
CONSOLE:  ng!
CONSOLE:  SYS: Build 89:54fa51cb6ea0 (2014.01.0) CLEAN
CONSOLE:  SYS: I am 0x870728575049725066
CONSOLE:  EFF52 and I have 128KB of flash!
CONSOLE:  IMU: MPU6050 I2C connection [OK].
CONSOLE:  IMU: HMC5883 I2C connection [OK].
CONSOLE:  IMU: MS5611 I2C connection [OK].
CONSOLE:  MPU6050: Self test [OK].
CONSOLE:  HMC5883L: Self test [OK].
CONSOLE:  MS5611: Self test [OK].

starting telemetry
```

I think it's related to #15, it seems to be connected the green light is up on the flie and radio.

Any ideas? Any ideas how to debug this?

Thanks

**Comment by tikot** [comment]
Also I think this is related to this error.

``` shell
{ [Error: **UNKNOWN**] errno: 5 }
{ [Error: **UNKNOWN**] errno: 5 }
```

**Comment by duanearnett** [comment]
@tikot I'm also seeing this issue. Did you figure anything out? I'm new to the CF and trying to track down if I'm doing something wrong or not. 
If I just run the script (the simple example from the docs) it will connect and hang at the "starting telemetry" line. If I restart the CF and the script I will get the log output below:

``` radio://1/80/250KPS
Using copter at radio://1/80/250KPS
STATE: setup --> connected
starting telemetry
CONSOLE: SYS: -------------------------
CONSOLE: ng!

CONSOLE: SYS: Build 164:0dbbb77772a9 (p
CONSOLE: roduction1.1) CLEAN

CONSOLE: SYS: I am 0x31323832313347032C
CONSOLE: 0031 and I have 1024KB of flas
CONSOLE: h!

CONSOLE: IMU: MPU9250 I2C connection [O
CONSOLE: K].

CONSOLE: IMU: AK8963 I2C connection [OK
CONSOLE: ].

CONSOLE: IMU: LPS25H I2C connection [OK
CONSOLE: ].

CONSOLE: EXPBRD: Found 0 memories.

CONSOLE: EEPROM: I2C connection [OK].

CONSOLE: AK8963: Self test [OK].

CONSOLE: LPS25H: Self test [OK].

CONSOLE: OW: Found 0.

CONSOLE: SYS: Free heap: 6048 bytes

```

Is the library, the CF, or myself defective?

**Comment by tikot** [comment]
@duanearnett I did found some interesting facts. I just got my CF2 will be doing some more testing, but I think to solve some issues is to write a C++ library that will compile a node module (`.node`).

My findings:
If you put crazyflie really close to the radio (USB dongle) it will connect (at least mine CF does that) which brings me to thinking it could be a power issue the radio does not get enough power supply. I don't have a power meter to test this theory.

It could be that there is an interference with communication between CF and radio. Changes the channels did help but then it failed again. Did testing with this idea couldn't come to a conclusion. Maybe some wireless thing is interfering with it.

Next it could be the issues with radio itself... 

Hoping to get some answers with CF2. Still want to play with C++ library and see what will happen. If you are curious what I did with aerogel, take a look at the project [autoflie](https://github.com/tikot/autoflie). You might find some answers.
